robotino_bringup/robots/tuw-robotino2/urdf/robotino.urdf

```diff
@@ -334,7 +334,7 @@
   </gazebo>
   <joint name="hokuyo_joint" type="fixed">
     <origin rpy="0.0 0.0 -0.015" xyz="0.184 0.004 0.16"/>
-    <parent link="base_linkz"/>
+    <parent link="base_link"/>
```

Comment:
I think this is wrong... the robotino.urdf is the file used by moveit!, doesn't it?